core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_body_directive.html

```diff
@@ -131,4 +136,25 @@
     text-align: center;
     text-shadow: -1px 0 white, 0 1px white, 1px 0 white, 0 -1px white;
   }
+
+  simple-editor-body .question-type-wrapper {
+    background-color: #fff;
+    border-radius: 4px;
+    box-shadow: 0 10px 20px rgba(0,0,0,0.19), 0 6px 6px rgba(0,0,0,0.23);
```

Comment:
They are vertical  and horizontal shadow, i think it'll be good to use material design shadow. https://codepen.io/sdthornton/pen/wBZdXq